packages/trader/src/App/Components/Elements/market-countdown-timer.jsx

```diff
@@ -0,0 +1,156 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { WS } from 'Services/ws-methods';
+import { connect } from 'Stores/connect';
+
+// check market in coming 7 days
+const days_to_check_before_exit = 7;
+
+const getTradingTimes = async target_time => {
+    return new Promise(resolve => {
+        WS.tradingTimes(target_time).then(data => {
+            if (data.error) resolve({ api_initial_load_error: data.error.message });
+            resolve(data);
+        });
+    });
```

Comment:
Since there is no reject handled here, can we refactor the above function to: 
```js
const data = await WS.tradingTimes(target_time);
if (data.error) {
  return {
    api_initial_load_error: data.error.message 
  };
}
return data;
}
```